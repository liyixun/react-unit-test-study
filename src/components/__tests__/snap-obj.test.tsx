import {} from "@testing-library/react";
describe("测试定制对象的快照内容", () => {
  it("正确匹配快照", () => {
    const user = {
      name: "eason",
      age: 18,
    };
    expect(user).toMatchSnapshot({
      name: "eason",
      age: 18,
    });
  });
});
