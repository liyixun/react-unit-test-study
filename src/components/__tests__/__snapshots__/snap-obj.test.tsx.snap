// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`测试定制对象的快照内容 正确匹配快照 1`] = `
Object {
  "age": 18,
  "name": "eason",
}
`;
